backends-clickhouse/src/main/scala/org/apache/spark/sql/execution/datasources/utils/MergeTreePartsPartitionsUtil.scala

```diff
@@ -27,13 +29,60 @@ import org.apache.spark.sql.connector.read.InputPartition
 import org.apache.spark.sql.execution.datasources.{HadoopFsRelation, PartitionDirectory}
 import org.apache.spark.sql.execution.datasources.v2.clickhouse.metadata.AddMergeTreeParts
 import org.apache.spark.sql.execution.datasources.v2.clickhouse.table.ClickHouseTableV2
+import org.apache.spark.sql.types.StructType
 import org.apache.spark.util.SparkResourceUtil
 import org.apache.spark.util.collection.BitSet
 
+import io.substrait.proto.{NamedStruct, Type}
+
+import scala.collection.JavaConverters._
 import scala.collection.mutable.ArrayBuffer
 
+// scalastyle:off argcount
 object MergeTreePartsPartitionsUtil extends Logging {
 
+  val DEFAULT_ORDER_BY_KEY = "tuple()"
+
+  def genOrderByAndPrimaryKeyStr(
```

Comment:
irrelevant to "MergeTreePartsPartitionsUtil"

Reply:
move to `MergeTreeDeltaUtil`